### DISCUSSION TITLE
`operationsSorter` for API Reference

### DISCUSSION THREAD

**Discussion by inyourtime** [scalar/scalar#3378] [Ideas]
### Description:

I would like to request the addition of an `operationsSorter` feature to the API reference. This feature would allow users to customize the order in which operations or endpoint (we already have `tagsSorter`) are displayed, improving usability and navigation.

### Use Case:

- Enhanced Clarity: Users could sort operations based on their specific needs (e.g., by HTTP method, alphabetical order, etc.), making it easier to locate relevant endpoints.
- Improved Documentation: A customizable sorter can lead to a more organized and user-friendly documentation experience.

### Proposed Implementation:

- Introduce an `operationsSorter` parameter in the API reference configuration.
- Support various sorting methods, such as: Alphabetical, By HTTP method (GET, POST, etc.), Custom user-defined sorting

So, I can submit a PR for this.

**Comment by marclave** [comment]
i dig it! lets do it, feel free to implement it @inyourtime  :)
